workflows/microbiome/README.md

```diff
@@ -0,0 +1,89 @@
+# Microbiome Workflows
+
+The following workflows can be used directly for microbiome data analysis, pathogen detection, and tracking purposes. The workflows can also be adapted to any other sequencing technique.
+
+To learn more about the following workflows and try them with real datasets, please check out our Microbiome tutorials on the Galaxy Training Network [GTN](https://training.galaxyproject.org/training-material/topics/microbiome/)
+
+## Nanopore _Preprocessing_
```

Comment:
Why italicize just Preprocessing here ? This looks a little odd

Reply:
I was trying to relate to the workflows' names in the Foodborne paper since the paper I will publish has just the name Preprocesing, not Nanopore Preprocessing. 

But true here it looks very odd, and I cannot really mention the paper here since this directory should be general enough for later to add any microbiome related workflow not just for foodborne

Now I removed all italics